README.md

```diff
@@ -143,6 +178,134 @@ has_audit_log enqueue_with: { priority: 30 }
 # Or for custom journaling:
 journal_attributes :email, enqueue_with: { priority: 20, queue: 'journaled' }
 ```
+##### Database-Backed Event Processing (Optional)
+
+Journaled includes a built-in database-backed delivery adapter with horizontally scalable workers. This is useful if you want to:
+
+- **Decouple from Delayed Job/ActiveJob**: Process events independently of your existing job queue
+- **Horizontal scaling**: Run multiple worker daemons across different servers/containers
+- **Batch API efficiency**: Use Kinesis `put_records` (batch) API instead of `put_record` (single), reducing API calls by up to 500x
+- **Custom retry logic**: Fine-tune retry behavior and failed event handling
+- **Visibility**: Events are stored in database tables where you can inspect, query, and monitor them
```

Comment:
IMO, this dedicated worker / outbox pattern should become the only offered strategy in the next major release of journaled, once we've refined it & rolled it out everywhere. So I might cut out this entire pitch because it's not something we should have to sell anyone on -- it'll just be the way journaled works.